### ISSUE TITLE
endpoint group gui not work

### ISSUE THREAD

**Issue by akihirobessy** [dOpensource/dsiprouter#244]
When I enter the information from Endpoint Groups gui and press the add button, it doesn't work.
In the first place, there is data displayed as Loading... in the initial display state. 
And If you try to update that data, it will not be reflected.
It seems that the Endpoint Groups screen have no response for update.

I started the debian9 strech server on ec2@AWS and installed dsiprouter v0.621 successfully as documented
and configured the load balancer with ALB with https.

![image](https://user-images.githubusercontent.com/20277168/87237298-8c9fa080-c42f-11ea-9e08-9eedbe2b670f.png)

**Comment by mackhendricks** [comment]
We will try to reproduce and let you know what we find out.

**Comment by akihirobessy** [comment]
Thank you for reply.

When it was happen, I just only installed dsiprouter on debian9 and changed nothing.
My Chrome dev tools showed nothing.
The systemctl status of dsiprouter, kamailio, rtpengine, mysql shows normal.

If you give me how to make debug level up in python or dsiprouter or anything, I will can check it detail.

**Comment by akihirobessy** [comment]
I solved my issue.

The reason is that I used port forwarding https 443 -> 5000 by ALB as AWS loadbalancer.
Because I'd like to let me access https by 443 port.

I changed to https 5000 -> 5000 so it worked.
It seems that the dsiprouter must use port number 5000 as https.

I have other question.
I ignored that let'sencrypt installing was failed when initial provisioning because I will use tls cert with ALB.

When dsiprouter working, it is no problem that my server can't use let'sencrypt?
